### PR TITLE
Update discouraged.md

### PR DESCRIPTION
**Pull request by pczarn2**
Added int to discouraged functions.

### CODE REVIEW

File: guides/en/discouraged.md [hunk 1]
```diff
@@ -102,9 +102,12 @@ The use of tags generating UI is generally discouraged by the CFML community. Se
 
 In almost all cases evaluate is unnecessary (use bracket notation instead). Use of evaluate can lead to performance and security issues.
 
+`int`
+Adobe recommends that you use the Floor function, instead of the Int function, in all new applications.See: https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-in-k/int.html
```
Comment (KamasamaK) [inline guides/en/discouraged.md:106]: Missing a space after the period

**Comment by pfreitag** [comment]
@pczarn2 two things on this: 

1) Do you know why Adobe recommends using floor instead of int? It would be useful to know the reason, for example does it support larger numbers than int? 

2) The discouraged markdown is generated automatically by adding a discouraged flag on the function or tag, rather than editing this file.

**Comment by pfreitag** [comment]
Ah - I see you have done this in #1610

**Comment by KamasamaK** [comment]
@pczarn2 If you're autogenerating this using `/utilities/guide.cfm` then this shouldn't require a manual review. I think this is just waiting on #1610.

**Comment by zspitzer** [comment]
As a heads up, Lucee already internally just uses `Math.floor()` for `int()`

https://github.com/lucee/Lucee/blob/5.3/core/src/main/java/lucee/runtime/functions/math/Int.java#L36

**Comment by pczarn2** [comment]
Closing, since the discouraged markdown is generated automatically by adding a discouraged flag on the function or tag, rather than editing this file.

**Pull request closed without merging**